### PR TITLE
[71] Adding investors to `CbWrangler` class

### LINKED ISSUE #71: Add investors to `CrunchbaseWrangler`
Add functionality to fetch investor organisations, and key people associated with the organisations

### PR DESCRIPTION
**Pull request by beingkk**
Closes #71 

Adds investor data and industry tags to the `CrunchbaseWrangler` class

The usage for fetching investor data:

```python
from innovation_sweet_spots.analysis.wrangling_utils import CrunchbaseWrangler
from innovation_sweet_spots.getters import crunchbase as cb

CbWrangler = CrunchbaseWrangler()

# Fetch a test list of Crunchbase organisations 
cb_orgs = cb.get_crunchbase_orgs(nrows=100)

cb_org_investors = CbWrangler.get_organisation_investors(cb_orgs)
```

cb_orgs is a dataframe that must have CB id and name columns
cb_org_investors will contain data about funding rounds and the corresponding `investor_name` and `investor_id` 

---

Checklist:

- [x] I have refactored my code out from `notebooks/`
- [x] I have checked the code runs
- [x] I have tested the code
- [x] I have run `pre-commit` and addressed any issues not automatically fixed
- [x] I have merged any new changes from `dev`
- [x] I have documented the code
  - [x] Major functions have docstrings
  - [ ] Appropriate information has been added to `README`s
- [x] I have explained the feature in this PR or (better) in `output/reports/`
- [x] I have requested a code review

### CODE REVIEW

**Review by Jack-0-0 — APPROVED** [review]
`CbWrangler.get_organisation_investors` runs as expected and the code looks good to merge. Thanks!

File: innovation_sweet_spots/analysis/wrangling_utils.py [hunk 1]
```diff
@@ -269,6 +269,62 @@ def convert_deal_currency_to_gbp(
...
             df[converted_column] = df[amount_column].copy()
         return df
+      
+    def get_funding_round_investors(self, funding_rounds: pd.DataFrame) -> pd.DataFrame:
+        """
+        Gets the investors involved in the specified funding rounds
+
+        Args:
+            Dataframe with organisations specified by 'org_id' and 'name', and
+            all their funding rounds (deals). Some of the important columns include:
```
Comment (Jack-0-0) [inline innovation_sweet_spots/analysis/wrangling_utils.py]: I noticed after approving that the Args section here doesn't refer to the variable and the sentence isn't complete.

Comment (beingkk) [inline innovation_sweet_spots/analysis/wrangling_utils.py]:
```suggestion
            all their funding rounds (deals).
```

**Comment by beingkk** [comment]
Thank you very much, @Jack-0-0 !

**Pull request merged**